Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2.py

```diff
@@ -243,6 +243,67 @@ def create_vm_parameters(args, subscription_id):
     return vm
 
 
+def create_nic_parameters(args, subscription_id):
+    """
+    Construct the NIC object
+
+    Use the actual parameters passed to the 'azure-vm-create-nic' command
+    to build a nic object that will be sent in the body of the command's associated
+    API call.
+
+    parameter: (dict) args
+        Dictionary that contains the actual parameters that were passed to the
+        'azure-vm-create-nic' command
+
+    returns:
+        NIC Object
+    """
+    # Retrieve relevant command arguments
+    resource_group = args.get('resource_group')
+    location = args.get('nic_location')
+    address_assignment_method = args.get('address_assignment_method')
+    private_ip_address = args.get('private_ip_address')
+    network_security_group = args.get('network_security_group')
+    vnet_name = args.get('vnet_name')
+    subnet_name = args.get('subnet_name')
+    ip_config_name = args.get('ip_config_name')
+    subnet_id = f"/subscriptions/{subscription_id}/resourceGroups/"
+    subnet_id += f"{resource_group}/providers/Microsoft.Network/virtualNetworks/" \
+                 f"{vnet_name}/subnets/{subnet_name}"
```

Comment:
it could also be one long f-string wrapped with parenthesis - like in the second comment.